src/main/kotlin/calculator/Expression.kt

```diff
@@ -0,0 +1,55 @@
+package calculator
+
+class Expression(list: List<String>) {
+
+    var list = list
+        private set
+
+    fun compute(): Long {
+        return list.stream().mapToLong{ s -> verifyStringToLong(s) }.sum()
+    }
+
+    private fun verifyStringToLong(input: String): Long {
+        var toLong: Long
+
+        try {
+            toLong = input.toLong()
+        } catch (e : NumberFormatException) {
+            throw IllegalArgumentException("수식에 문자(${input})가 들어올 수 없습니다.")
+        }
```

Comment:
코틀린에서는 `try`, `if`등의 결과도 바로 변수에 담을 수 있어요! 그렇게 하면 `val`을 사용할 수 있겠죠?
```suggestion
        val toLong: Long = try {
            input.toLong()
        } catch (e : NumberFormatException) {
            throw IllegalArgumentException("수식에 문자(${input})가 들어올 수 없습니다.")
        }
```

Reply:
넵!